app/src/main/kotlin/com/simplemobiletools/camera/activities/MainActivity.kt

```diff
@@ -435,6 +527,29 @@ class MainActivity : SimpleActivity(), PhotoProcessor.MediaSavedListener {
         })
     }
 
+    // TODO: May want to put this in CameraPreview.kt; next to tryTakePicture()
+    private fun tryTakeDelayedPicture() {
```

Comment:
Like you said, I think it belongs in CameraPreview.kt. I recommend doing it before merging if it isn't too much work. Or else, I'll make a task that can be completed during sprint 3.